### PR TITLE
AUI-55 Bye bye cocoapods hello Swift packages

### LINKED ISSUE #3: Unable to transition from CocoaPods to Swift Package
The BundleFileUpdater uses a Run Script the user has to configure in the project's Build Phases settings (cf. [README](https://github.com/apploft/BundleFileUpdater/#update-files-with-every-build)).

The following part of this script gets the file 'BundleFileUpdater.swift' from the disk:
`cat "$PODS_ROOT/BundleFileUpdater/BundleFileUpdater/BundleFileUpdater.swift"`

Then environment variable `$PODS_ROOT` is used here to get the path to the directory of this library. There is no Swift Package equivalent environment variable to get the path to the directory of the packages.

A possible solution:
Users have to copy the file 'BundleFileUpdater.swift' into their project and enter the path into the script using the environment variable `$SRCROOT`, which is the path to the project directory.

@tinora and I discussed this solution and came to the conclusion that the BundleFileUpdater will currently not be updated to a Swift Package since the solution is not good enough. The BundleFileUpdater will be updated once there is a proper solution for this problem is found, or it remains as one of the only used intern libraries using CocoaPods.

In Apple's Swift feedback forum I have created an associated [task](https://bugs.swift.org/browse/SR-12979) & I asked on [Stackoverflow an according question to this problem](https://stackoverflow.com/questions/62322628/cocoapods-pods-root-environment-variable-equivalent-in-swift-packages)

### PR DESCRIPTION
**Pull request by Ali-Pourasad**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "BundleFileUpdater",
+    products: [
```
Comment (tinora) [inline Package.swift:11]: @Ali-Pourasad please add the 'platform' specification and specify 'iOS(.v12)'

File: README.md [hunk 2]
```diff
@@ -48,12 +47,9 @@ cat "$PODS_ROOT/BundleFileUpdater/BundleFileUpdater/BundleFileUpdater.swift" "$S
...
 ## Installation
 
-BundleFileUpdater is available through [CocoaPods](http://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+### [Swift Package](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md#supportedplatform)
 
-```ruby
-pod "BundleFileUpdater"
-```
+Just integrate the APLNetworkLayer via Xcode 11 - that's it!
```
Comment (tinora) [inline README.md]: @Ali-Pourasad what has APLNetworkLayer to do here? ;)

**Comment by markusfassbender-parshipgroup** [comment]
The merge doesn't show up on the public master branch, but as tag `1.0.0`. Maybe something went wrong?

**Comment by Ali-Pourasad** [comment]
The merge of this branch on the master was reverted, due to Issues described here: 
https://github.com/apploft/BundleFileUpdater/issues/3

**Pull request merged**